code_generation/data/attribute_classes/input.json

```diff
@@ -494,6 +494,42 @@
                     "description": "line drop compensation reactance"
                 }
             ]
+        },
+        {
+            "name": "GenericCurrentSensorInput",
+            "base": "SensorInput",
+            "attributes": [
+                {
+                    "data_type": "MeasuredTerminalType",
+                    "names": "measured_terminal_type",
+                    "description": "type of measured terminal"
+                },
+                {
+                    "data_type": "AngleMeasurementType",
+                    "names": "angle_measurement_type",
+                    "description": "type of angle measurement"
+                },
+                {
+                    "data_type": "double",
+                    "names": "i_sigma",
```

Comment:
TODO: determine the right way to formulate sigma's for a current sensor.
Multiple options to investigate:

- just i_sigma - similar as for voltage sensor - chosen now
- i_sigma and i_angle_sigma as double - similar as power_sigma
- i_sigma and i_angle_sigma as RealValue<sym> - similar as p_sigma and q_sigma

Reply:
After discussing with @TonyXiang8787: `i_sigma` and `i_angle_sigma` as a double, where `i_angle_sigma` is Optional.